.storybook/3.COLORS.stories.mdx

```diff
@@ -84,37 +85,35 @@ var(--color-primary-alternative)
 var(--color-primary-muted)
 var(--color-primary-inverse)
 
-var(--color-secondary-default) [DEPRECATED]
-var(--color-secondary-alternative) [DEPRECATED]
-var(--color-secondary-muted) [DEPRECATED]
-var(--color-secondary-inverse) [DEPRECATED]
-var(--color-secondary-disabled) [DEPRECATED]
```

Comment:
Removing deprecated secondary colors which will be removed in v4, may need to rebase if this is merged first https://github.com/MetaMask/metamask-extension/pull/24971